### PR TITLE
Examples: Render matplotlib interactively with plotly

### PR DESCRIPTION
**Pull request by lgeiger**
Just discovered that plotly can render interactive matplotlib figures! 🎉 

That's amazing 📊

### CODE REVIEW

**Comment by BenRussert** [comment]
Nice!

File: docs/Usage/Examples.md [hunk 1]
```diff
@@ -5,11 +5,23 @@ Please share your favorite snippets with us and add them to this page.
...
 from plotly import offline
 offline.init_notebook_mode()
 
 offline.iplot([{"y": [1, 2, 1]}])
+{%- language name="Python using matplotlib", type="py" -%}
+import numpy as np
+import matplotlib.pyplot as plt
+from plotly import offline as py
+import plotly.tools as tls
+py.init_notebook_mode()
```
Comment (rgbkrk) [inline docs/Usage/Examples.md]: I so wish we didn't have to do this since we're using our packaged Plotly, alas it will warn otherwise.

Comment (lgeiger) [inline docs/Usage/Examples.md]: Yeah! That would be excellent

Comment (rgbkrk) [inline docs/Usage/Examples.md]: /cc @aseier :wink:

**Pull request merged**